app/models/ranking_entry.rb

```diff
@@ -0,0 +1,70 @@
+# Smallest unit of a ranking list. Contains different data for one team and contains logic to parse those date out of
+# match objects.
+class RankingEntry
+  attr_reader :rank, :name, :match_count, :won_count, :draw_count, :lost_count, :goals, :goals_against, :goals_difference, :points
+  attr_accessor :rank
```

Comment:
To my mind, the `attr_accessor :rank` makes the `attr_reader :rank` redundant since `attr_accessor` includes an attribute reader as well as a writer. See http://ruby-doc.org/core-2.0.0/Module.html#method-i-attr_accessor